frontend/src/common/queries/wheresMyGene.ts

```diff
@@ -807,7 +807,11 @@ function addCellTypeRowToResult({
      * (thuang): We manually indent 4 spaces instead of using CSS, so we don't
      * have to update SVG render function to mimic CSS padding
      */
-    const name = isAggregated ? rawName : `    ${rawName || "unknown"}`;
+    const name = isAggregated
+      ? rawName
+      : compareOptionId === COMPARE_OPTION_ID_FOR_UNKNOWN
+      ? `    ${rawName || COMPARE_OPTION_ID_FOR_UNKNOWN}`
```

Comment:
why is this needed again lol? Sorry I don't have the context 😆 Thank you!

Reply:
![image](https://user-images.githubusercontent.com/109984998/225991271-f641d1f4-6c0b-4dcd-ab03-d4bb81b933e2.png)

multiethnic is being set to "unknown" because the rawName is null from the BE response, similar to how the unknown compareOptionId is null for the name

Reply:
ah I thought you changed BE to use id as name already. Did we lose that in merge conflict resolution or something?

Reply:
hmmm you're right, I remember the conversation now

I'll take a look into it

Reply:
I think [this commit](https://github.com/chanzuckerberg/single-cell-data-portal/pull/4202/commits/a40bed6d2c82ae2410278855e8d4dcc9cc543cb0) from @atarashansky is what is passing multiethnic `name` back as null from the BE

Should we keep the name as null as it is or return the compare option value for `name` as before?

Reply:
I vote for returning the id as `name` since `null` is pretty useless 🙏

Reply:
Alec is working on a patch for this now!